txmongo/collection.py

```diff
@@ -2,7 +2,7 @@
 # Use of this source code is governed by the Apache License that can be
 # found in the LICENSE file.
 
-import types
+from __future__ import absolute_import, division
```

Comment:
I'm going off the Twisted py3 porting standards (https://twistedmatrix.com/trac/wiki/Plan/Python3#Reviewerchecklist) where it's in there regardless. If that's an issue, I can change it.